public/script.js

```diff
@@ -6,6 +6,38 @@ $( document ).ready(function() {
   canvas.height = window.innerHeight;
   drawStuff();
 
+  $("#download").click(function() {
+    var outputCanvas = document.createElement("canvas");
+    outputCanvas.width = canvas.width;
+    outputCanvas.height = canvas.height;
+
+    var outputContext = outputCanvas.getContext("2d");
+    outputContext.font = "bold 24pt Times";
+
+    var gradient = outputContext.createLinearGradient(0, 0, canvas.width, 0);
+    gradient.addColorStop(0, "#ECF9FF");
```

Comment:
Hmm, so the initial goal was to change the background gradient on a daily basis. I never ended up automating that but the colours are now defined in index.js as `background_colours` - it'd be good to avoid having to define those colours twice. 

Additionally, I've been changing the angle of the background gradient as well as the colours - this won't be able to do that, will it? 

Perhaps it's possible to have the gradient background on the canvas all along?